packages/components/src/List/Toolbar/SelectAll/__snapshots__/SelectAll.snapshot.test.js.snap

```diff
@@ -2,74 +2,52 @@
 
 exports[`SelectAll should be unchecked when there is no items 1`] = `
 <form
-  className="theme-container navbar-form navbar-left"
+  className="theme-container"
 >
-  <div
-    className="checkbox-inline navbar-text theme-tc-list-toolbar-select-all"
-  >
-    <label
-      className="tc-list-toolbar-select-all"
-    >
-      <input
-        checked={false}
-        disabled={true}
-        onChange={[Function]}
-        type="checkbox"
-      />
-      <span>
-        Select all
-      </span>
-    </label>
-  </div>
+  <label>
+    <span
```

Comment:
if you want a better snap you should use a jest.unmock('@talend/design-system')` at the begining of the test